### PR TITLE
Refactor/sprint1 cleanup

### PR DESCRIPTION
**Pull request by GregoryForrest**
Some cleanup actions from Sprint 1
1. Removed several relative references
2. Removed a carryover Errors dialog and utilities file from TMCR that isn't being used
3. Better load Program Names and the associated PECs by caching the transformation, and prevent user typo from breaking the tool by JSON.parse error not being caught and handled
4. Embed the PeoplePicker inside it's own RHF -- Provide ability to use it in the DEV environment by providing some sample users
5. Provide error if no user selected in PopupPeoplePicker

### CODE REVIEW

File: src/components/PeoplePicker/PeoplePicker.tsx [hunk 1]
```diff
@@ -13,6 +13,7 @@ import { LayerHost } from "@fluentui/react";
 import { FieldValues, useController } from "react-hook-form";
 import { BaseFormField } from "components/BaseFormFields/BaseTypeDef";
 import { TextFieldIcon } from "@fluentui/react-icons-mdl2";
+import { spPeoplePickerData } from "../../api/SPSampleUserData";
```
Comment (RobertPorterfield) [inline src/components/PeoplePicker/PeoplePicker.tsx]: Relative import...

Comment (GregoryForrest) [reply src/components/PeoplePicker/PeoplePicker.tsx]: Good catch!  Fixed!

File: src/components/PeoplePicker/PopupPeoplePicker.tsx [hunk 2]
```diff
@@ -7,125 +7,113 @@ import {
...
-                          Id: 1,
-                          Title: "FORREST, GREGORY M CIV USAF AFMC AFLCMC/OZIC",
-                          EMail: "gregory.forrest.1@us.af.mil",
-                          text: "FORREST, GREGORY M CIV USAF AFMC AFLCMC/OZIC",
-                          secondaryText: "",
-                        },
-                      ]);
-                    }
+                    onUpdate([currentUser]);
+                    setOpen(false);
```
Comment (RobertPorterfield) [inline src/components/PeoplePicker/PopupPeoplePicker.tsx:105]: Nice change! Very clean 👍

**Review by RobertPorterfield — APPROVED** [review]
Nice job with some cleanup!

**Pull request merged**